manifests/role/hathitrust/dev/app_host_and_ruby.pp

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Grog
+class nebula::role::hathitrust::dev::app_host_and_ruby {
```

Comment:
The dev::app_host role only applies to ht-web-testing and (until this change is applied) grog. I don't know how ht-web-testing is an "App Host" when it lacks ruby.

Alternatively dev::app_host could get nebula::profile::ruby and we would add a dev::app_host_without_ruby or similar.

Reply:
Got rid of the `_and_ruby` by removing ht-web-testing's requirement entirely.